lib/itc/post_data.rb

```diff
@@ -107,6 +107,32 @@ def update_app_data(config, localization)
           isEmptyValue: true,
           isRequired: false
         },
+        watchAppIcon: {
+          errorKeys: nil,
+          isEditable: true,
+          isRequired: false,
+          value: {
+            assetToken: nil,
+            originalFileName: nil,
+            sortOrder: nil,
+            thumbNailUrl: nil,
+            thumbnailAssetToken: nil,
+            url: nil
+          }
+        },
+        bundleInfo: {
```

Comment:
@JEnneverDWNLD - I added `bundleInfo` and the params below because I noticed it was in the example POST body from https://github.com/fastlane/itc-api-docs/blob/master/examples/update_request.md and when submitting from the itunesconnect site but am not sure if they're necessary. Have you seen these params before and omitted them?